### DISCUSSION TITLE
Can I install without Docker and Snap on an RPI?

### DISCUSSION THREAD

**Discussion by Mastiffen** [zwave-js/zwave-js-ui#1350] [Q&A]
I followed the instructions on the quick start guide, but ran in to a snag:

`cd ~
mkdir zwavejs2mqtt
cd zwavejs2mqtt
# download latest version
curl -s https://api.github.com/repos/zwave-js/zwavejs2mqtt/releases/latest  \
| grep "browser_download_url.*zip" \
| cut -d : -f 2,3 \
| tr -d \" \
| wget -i -
unzip zwavejs2mqtt-v*.zip
./zwavejs2mqtt`

It gave me an error message (translated from Norwegian, so it's probably not word by word what I would get on an English version of Rasberry Pi OS):

`bash: ./zwavejs2mqtt: could not run binary file: Invalid format on executable file`

Can somebody please tell me what I'm doing wrong?

**Comment by robertsLando** [comment]
@Mastiffen The zip file on release is for amd64 arch, rpi is armv7. I suggest you to install nodejs and git on your rpi and do a manual build if you don't want to use docker or snap:

```bash
git clone https://github.com/zwave-js/zwavejs2mqtt
cd zwavejs2mqtt
yarn install
yarn run build
yarn start
```

**Reply by Mastiffen** [reply]
I got it working, thanks!
